tests/test_openms.py

```
@@ -37,3 +43,14 @@ def test_get_openms_file_name(input_file, expected_file, extension):
 def test_parse_tolerence(input_str, expected_tol, expected_unit):
     assert parse_tolerance(input_str) == (expected_tol, expected_unit)
+
+
+@pytest.mark.parametrize("plex_name", TMT_PLEXES)
+def test_tmt_label_inference_full_plexes(plex_name):
+    assert plex_name == infer_tmtplex(TMT_PLEXES[plex_name])
+
+
+@pytest.mark.parametrize("plex_name", TMT_PLEXES)
+def test_tmt_label_inference_from_incomplete_plexes(plex_name):
+    labels = {**TMT_PLEXES[plex_name]}
```

Ruff (0.8.2)

Local variable `labels` is assigned to but never used

Remove assignment to unused variable `labels`

(F841)